internal/generator/vector/helpers/helpers.go

```diff
@@ -63,3 +66,17 @@ func ConfigPath(name string, file string) string {
 func SecretPath(secretName string, file string) string {
 	return fmt.Sprintf("%q", filepath.Join("/var/run/ocp-collector/secrets", secretName, file))
 }
+
+func FromVectorSecretWithSecretPath(secretName, key string) string {
```

Comment:
This name is really long and not accurate when viewing the actual config. Consider:

```
helpers.SecretFrom(secretKey) or helpers.SecretConfigFrom(secretKey) 
```
or if we have a secrets package:
```
secrets.ConfigFrom(secretKey)
```